chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -45,33 +48,37 @@
     private static final String HAS_NON_CONFORMITY = "Non-Conformity";
 
     @Autowired
-    SurveillanceDAO survDao;
+    private SurveillanceDAO survDao;
     @Autowired
-    CertifiedProductDAO cpDao;
+    private CertifiedProductDAO cpDao;
     @Autowired
-    CertificationResultDetailsDAO certResultDetailsDao;;
+    private CertificationResultDetailsDAO certResultDetailsDao;;
     @Autowired
-    CertificationCriterionDAO criterionDao;
+    private CertificationCriterionDAO criterionDao;
     @Autowired
-    UserPermissionDAO userPermissionDao;
+    private MessageSource messageSource;
     @Autowired
-    MessageSource messageSource;
+    private ErrorMessageUtil msgUtil;
 
-    public void validate(Surveillance surv) {
+    /**
+     * Validate a surveillance.
+     * @param surv the surveillance to validate
+     */
+    public void validate(final Surveillance surv) {
         CertifiedProductDetailsDTO cpDetails = null;
 
         // make sure chpl id is valid
         if (surv.getCertifiedProduct() == null) {
             surv.getErrorMessages()
-                    .add(messageSource.getMessage(
-                            new DefaultMessageSourceResolvable("surveillance.nullCertifiedProduct"),
-                            LocaleContextHolder.getLocale()));
+            .add(messageSource.getMessage(
```

Comment:
Can you convert this to use msgUtil and remove messageSource as a variable in this class?